etc/kayobe/kolla.yml

```diff
@@ -565,7 +565,7 @@ kolla_build_customizations: "{{ kolla_build_customizations_common | combine(koll
 #kolla_enable_openvswitch:
 #kolla_enable_osprofiler:
 #kolla_enable_outward_rabbitmq:
-#kolla_enable_ovn:
+kolla_enable_ovn: yes
```

Comment:
We can now remove this flag from these files:

* `etc/kayobe/environments/aufn-ceph/kolla.yml`
* `etc/kayobe/environments/ci-builder/stackhpc-ci.yml`
* `etc/kayobe/environments/ci-multinode/kolla.yml`

Reply:
and the other two?

Reply:
This branch was created from master, I didn't notice PR was targeting Yoga by default.